boards/arm/arduino_portenta_h7/arduino_portenta_h7_m4.yaml

```diff
@@ -0,0 +1,18 @@
+identifier: arduino_portenta_h7_m4
+name: Arduino Portenta H7
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+ram: 288
+flash: 1024
+supported:
+  - arduino_gpio
```

Comment:
`arduino_gpio` represents the usual arduino uno connector which is not present here. Please remove.

Reply:
For the longer term, a arduino_mkr connecter device tree binding could be created but not needed for this PR.